### ISSUE TITLE
Package structure

### ISSUE THREAD

**Issue by cmaumet** [incf-nidash/nidmresults-fslhtml#15]
For better compliance with Python conventions, we should reorganize the repository. For consistency with [pyNIDM](https://github.com/incf-nidash/PyNIDM), I would suggest:

```
nidmfslhtml/queries
           /nidmviewerfsl
           /nidmviewerfsl/tests
           /bin
           /style
setup.py
requirements.txt
```

**Comment by TomMaullin** [comment]
I'm glad you've suggested that! Only just this morning I started a branch doing exactly this and addressing PEP-8 formatting!

**Comment by TomMaullin** [comment]
This has been addressed in #16
